nestjs/src/courses/mentors/mentors.controller.ts

```diff
@@ -22,4 +23,21 @@ export class MentorsController {
     const items = await this.mentorsService.getStudents(mentorId);
     return items.map(item => new MentorStudentDto(item));
   }
+
+  @Get('/:mentorId/dashboard/:courseId')
+  @UseGuards(DefaultGuard)
```

Comment:
consider using `@UseGuards(DefaultGuard)` on scope of controller to avoid duplication

Reply:
Ok, thanks. It is my first experience with nest, so most of the things are just repeating from somewhere. I will really grateful if you add more comments on BE side